src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -387,6 +387,11 @@ public function getConfiguration(array $config, ContainerBuilder $container)
 
     private function registerFormConfiguration(array $config, ContainerBuilder $container, XmlFileLoader $loader)
     {
+        if (false === $config['form']['improved_validation_messages']) {
+            @trigger_error('Setting the "framework.form.improved_validation_messages" to \'false\' has been deprecated and will be set to \'true\' by default in Symfony 5.0.', E_USER_DEPRECATED);
```

Comment:
I don't know if we can improve this sentence, but it will not only be `true` by default in 5.0; it'll always be true and previous messages dropped.
Perhaps:

> Setting the "framework.form.improved_validation_messages" to \'false\' has been deprecated and will have no effect as of Symfony 5.0.